examples/test-c-server.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * SRT - Secure, Reliable, Transport
+ * Copyright (c) 2017 Haivision Systems Inc.
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; If not, see <http://www.gnu.org/licenses/>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <arpa/inet.h>
+
+#include "srt.h"
+
+int main( int argc, char** argv )
+{
+    int ss, st;
+    struct sockaddr_in sa;
+    int yes = 1;
+    struct sockaddr_storage their_addr;
+
+    printf("srt startup\n");
+    srt_startup();
+
+    printf("srt socket\n");
+    ss = srt_socket(AF_INET, SOCK_DGRAM, 0);
+    if ( ss == SRT_ERROR )
+    {
+        fprintf(stderr, "srt_socket: %s\n", srt_getlasterror_str());
+        return 1;
+    }
+
+    printf("srt bind address\n");
+    sa.sin_port = htons(atoi(argv[2]));
+    if ( inet_pton(AF_INET, argv[1], &sa.sin_addr) != 1)
+    {
+        return 1;
+    }
+
+    printf("srt setsockflag\n");
+    srt_setsockflag(ss, SRTO_TWOWAYDATA, &yes, sizeof yes);
```

Comment:
The `SRTO_TWOWAYDATA` options should be already removed, or at least it's deprecated. This was an early, but abandoned, method of making SRT bidirectional. Version 1.2.0 is unidirectional and requires `SRTO_SENDER` flag to be set by the sending party, and version 1.3.0 is bidirectional (only of both sides are), it's bidirectional by default, and requires no extra flags for that.

Reply:
So, we can remove it... However, I feel it's here for explaining how to set flags, even if it's irrelevant.

Reply:
At least a different flag name should be used :)